README.md

```diff
@@ -40,7 +40,7 @@ For a detailed breakdown of motivations, see:
 ### Principles:
 
 - All Temporal objects are immutable.
-- Date values can be represented in local calendar systems, but they should be convertable to and from the [Proleptic Gregorian Calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar).
+- Date values can be represented in local calendar systems, but they should be convertable to and from the [Proleptic Gregorian Calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar). Read more on [non-Gregorian calendar motivation](./docs/calendar-review.md).
```

Comment:
This is the TL;DR of the entire proposal, so we should keep words to a minimum here.

```suggestion
- Date values can be represented in local calendar systems ([why?](./docs/calendar-review.md)), but they should be convertable to and from the [Proleptic Gregorian Calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar).
```